data/seeds/00-data.js

```diff
@@ -0,0 +1,20 @@
+const file = require('../incidents.json');
```

Comment:
From what we discussed, this seeding function does not populate the table. Let's work on it more